app/controllers/mixins/actions/vm_actions/reconfigure.rb

```diff
@@ -208,12 +229,24 @@ def get_reconfig_info(reconfigure_ids)
                         :cb_bootable => disk.bootable}
           end
 
+          # reconfiguring network adapters is only supported when one vm was selected
+          network_adapters = []
+          if @reconfigureitems.size == 1
+            vm = @reconfigureitems.first
+
+            vm.hardware.guest_devices.order(:device_name => 'asc').each do |guest_device|
+              lan = Lan.find_by(:id => guest_device.lan_id)
```

Comment:
Rbac?

Reply:
@himdel Is rbac necessary for lans that are already connected to the vm? Should the vm owner be blocked removing a non accessible lan from his vm?

Reply:
Oh, you're right, here it makes less sense, never mind :).

(But if you wanted to, it would be spelled `find_record_with_rbac(Lan, guest_device.lan_id)`.)